### ISSUE TITLE
Support isHidden flag on Windows

### ISSUE THREAD

**Issue by CharlieHess** [Teamwork/node-auto-launch#2]
With the registry approach, I'm not sure of a way to support `isHidden`. But could you append it to the launch path as a command-line argument? Something like this:
``` coffeescript
if isHidden
  regKey.set opts.appName, Winreg.REG_SZ, "#{opts.appPath} -hidden", cb
else
  # Same as before
```
That way, individual apps could parse the argument on their end to support it.

**Comment by 4ver** [comment]
Thanks for the suggestion! I had been meaning to do this alright but it's a bit annoying that I can't provide consistent behaviour for both windows and osx. Will look further into it asap.

**Comment by dpac4u** [comment]
is there any update on this. I am eagerly waiting for this fix. Thanks

**Comment by Aluxian** [comment]
Me too, I want this for my app: https://github.com/Aluxian/Facebook-Messenger-Desktop
Thanks!

**Comment by alexcroox** [comment]
+1

**Comment by yago66** [comment]
+1

**Comment by adam-lynch** [comment]
FYI this isn't being worked on as far as I know. Any insight / contributions welcome.

**Comment by adam-lynch** [comment]
Resolved by #26
